libc/src/math/generic/dfmaf128.cpp

```diff
@@ -0,0 +1,25 @@
+//===-- Implementation of dfmaf128 header -----------------------*- C++ -*-===//
```

Comment:
```suggestion
//===-- Implementation of dfmaf128 function -------------------------------===//
```

You can just copy-paste it, or even just click the "Commit suggestion" button on GitHub.